package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "hyperswitch",
-  "version": "0.8.8",
+  "version": "0.9.0",
```

Comment:
Semantic versioning is designed to communicate changes, independent of whether someone cares or not. Bumping a version here is very cheap (cheaper than this discussion), and given the special npm rules about versions < 1.0 should avoid people pulling in the newer version automatically.